src/mutations/system/singleFileUpload.ts

```diff
@@ -117,7 +117,17 @@ const resolver: MutationToSingleFileUploadResolver = async (
   }
 
   const uuid = v4()
-  const key = await systemService.aws.baseUploadFile(type, upload, uuid)
+  const settled = await Promise.allSettled([
```

Comment:
Not using Promise.all to require a successful uploading on both AWS and Cloudflare?